models/User.go

```diff
@@ -978,6 +1001,8 @@ func PlaceBidOrder(userId uint32, bid *Bid) (uint32, error) {
 	orderFee := getOrderFee(bid.StockQuantity, orderPrice)
 	cashLeft := int64(user.Cash) - int64(bid.StockQuantity*orderPrice+orderFee)
```

Comment:
We're declaring reservedCash variable below. Can we simply use that here instead of doing bid.StockQuantity*orderPrice. Makes it slightly more readable